cooltools/sandbox/expected_smoothing.py

```diff
@@ -268,6 +289,37 @@ def agg_smooth_cvd(
     -------
     cvd_smoothed : pandas.DataFrame
         A cvd table with extra column for the log-smoothed contact frequencies (by default, "balanced.avg.smoothed").
+
+    Notes
+    -----
```

Comment:
let's rethink this docstring @Yaoyx -- I think this one should focus on the key:column mapping for those in DEFAULT_CVD_COLS, and what is below is probably better for the expected_cis doctsring